common/protocol/Ola.proto

```diff
@@ -66,15 +66,15 @@ enum PluginIds {
   OLA_PLUGIN_UARTDMX = 20;
   OLA_PLUGIN_OPENPIXELCONTROL = 21;
   OLA_PLUGIN_GPIO = 22;
+  OLA_PLUGIN_SPIDMX = 10001;
```

Comment:
If we're merging to mainline this should have a non-dev code.

Reply:
Well spotted!

Reply:
Fixed.